File input output/Read file/task.md

```diff
@@ -1,11 +1,62 @@
 ## Read file
 
-Python has a number of built-in functions to read and write information from a file on your computer. The `open` function is used to open a file. The file can be opened in read mode (using `"r"` as the second argument) or in write mode (using `"w"` as the second argument). The `open` function returns the file object. You need to store it to close the file later.  
-  
-Print the contents of "input.txt" to output. Print the first line of "input1.txt". Then close the file.  
 
-Check out the section 7.2.1 "Methods of File Objects" in this [Python Tutorial chapter](https://docs.python.org/3/tutorial/inputoutput.html).
+- To read a file’s contents, you can call `f.read(size)`, which reads some quantity of data and returns it as 
+a string. When size is omitted or negative, the entire contents of the file will be read and returned.
 
-<div class='hint'>Use <code>print</code> function.</div>
-<div class='hint'>Use <code>readline()</code> method.</div>
-<div class='hint'>Use <code>f1.close()</code> method.</div>
+```python
+with open('somefile.txt') as f:
+    print(f.read())
+```
+```text
+Here's everything that's in the file.\n
+```
+<i>**Note**: there will be a problem if the file is twice as large as your machine’s memory.</i>
+
+
+- `f.readline()` reads a single line from the file; a newline character (`\n`) is left at the end of the 
```

Comment:
Again, not sure if need a list, it looks a bit funny with the snippet and the text interrupting it